Makefile

```diff
@@ -74,50 +100,37 @@ uninstall-rbac:
 	$(HELM) template $(STRING_SET_VALUES) -s templates/database_roles.yaml $(HELM_CHART) | kubectl delete -f -
 	$(HELM) template $(STRING_SET_VALUES) -s templates/operator_roles.yaml $(HELM_CHART) | kubectl delete -f -
 
+uninstall: manifests helm uninstall-chart uninstall-crd ## Uninstall CRDs from a cluster
 
-# Uninstall CRDs from a cluster
-uninstall: manifests helm uninstall-chart uninstall-crd
+##@ Deployment
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests helm install-chart install-crd
+deploy: manifests helm install-chart install-crd ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 
-# UnDeploy controller from the configured Kubernetes cluster in ~/.kube/config
-undeploy: uninstall-chart uninstall-crd
+undeploy: uninstall-chart uninstall-crd ## UnDeploy controller from the configured Kubernetes cluster in ~/.kube/config
 
-# Generate manifests e.g. CRD, RBAC etc.
-manifests: controller-gen
+manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=config/crd/bases
 	cp config/crd/bases/* $(HELM_CHART)/crds
 
-# Run go fmt against code
-fmt:
-	go fmt ./...
-
-# Run go vet against code
-vet:
-	go vet ./...
+##@ E2E
 
 # Run e2e tests locally using go build while also setting up a proxy in the shell to allow
```

Comment:
It did not feel correct to inline those 2 comments, hence I just left them where they are and added a new one below building on these